vuepress/docs/v6.3.2/docs/getting-started/README.md

```diff
@@ -106,7 +106,7 @@ multipass shell ubuntu-lts
 ::: tip Why K3s?
 K3s is a certified Kubernetes distribution designed for production workloads in resource-constrained environments.
 
-It's packaged as a single <40MB binary that reduces the dependencies and steps needed to install, run and auto-update a production Kubernetes cluster.
+It's packaged as a single <40MB binary that reduces the dependencies and steps needed to install, run, and auto-update a production Kubernetes cluster.
```

Comment:
I just noticed k3s now says <50MB. We could either change this to "small" or try to keep it sync and bump it to "50MB". I'd lean towards the latter. https://k3s.io/

Reply:
you got it :) i like using an absolute value as well; one of the items jinah and i are discussing for the style guide is when relative descriptors are appropriate